pulsequantum/livestream/sweepsettings.py

```diff
@@ -83,15 +87,19 @@ def get_settings(self):
         self.settings.fast_time = self.sequencebuilder.fast_time()
         self.settings.slow_steps = self.sequencebuilder.slow_steps()
         self.settings.marker_duration = self.sequencebuilder.marker_duration()
+        self.settings.delay_time = self.sequencebuilder.delay_time()
 
     def update_video(self):
-        self.video.alazarchansettings.settings.int_time = self.settings.fast_time*0.98 
-        self.video.alazarchansettings.settings.records_per_buffer = self.settings.slow_steps
-        if self.settings.scan_options == 'Sinusoidal':
-            self.video.alazarchansettings.settings.buffers_per_acquisition = self.settings.slow_steps
-            self.video.alazarchansettings.settings.integrate_samples = True
-            self.video.alazarchansettings.settings.int_time = self.settings.marker_duration
-        self.video.alazarchansettings.config()
+        try:
+            self.video.alazarchansettings.settings.int_time = self.settings.fast_time*0.98 
```

Comment:
I would like if you could use a dummy instrument instead of the allazar for testing

Reply:
We will do this later